naff/models/discord/message.py

```diff
@@ -380,6 +381,15 @@ def get_referenced_message(self) -> Optional["Message"]:
             return None
         return self._client.cache.get_message(self._channel_id, self._referenced_message_id)
 
+    def does_mention(self, query: "str | re.Pattern[str] | models.BaseUser | models.BaseChannel | models.Role") -> bool:
```

Comment:
*I'm open for other names for the method

Reply:
Let's call it `contains_mention`

Reply:
I'll change it as soon I'm on my computer